src/components/CircularImg/index.jsx

```diff
@@ -1,5 +1,7 @@
 import './circularimg.css'
 
-export default function CircularImg(props){
-    return (<img className="circular-img" src={props.href} alt={props.alt ? props.alt : ""} />)
+export default function CircularImg({href}){
+    return (
+        <img className="circular-img" src={href} alt="" />
```

Comment:
Could you still allow alt to be optionally passed?

Reply:
I deleted the  <ins>alt props</ins> because there was no property for it. <br>  You can add it aswell ! I should have thought you would use it later, that's my bad

Reply:
Yeah once we switch over to real images, we will be adding alts. If you could add it back that would be great. Was this throwing an error the original way it was done?

Reply:
Here is the error :   **4:71  error  'al 4:71  error  'alt' is missing in props validation**

Reply:
We are ignoring prop types in this change are we not?

Reply:
We are, but eslint is making this error by itself.
I've made the commit you wanted to change eslintrc & you will see by yourself. I'm leaving you the solution : https://stackoverflow.com/questions/38684925/react-eslint-error-missing-in-props-validation

Reply:
I completely understand about the prop-types getting ignored, please leave that in, but I am confused as to why leaving the alt the way it was is causing errors if we are disabling them?

Reply:
Here is a branch that keeps it the way it was originally, and doesn't throw errors anymore https://github.com/acm-uic/flourish-2023/tree/feature1